### ISSUE TITLE
Need ability to add legend for onset picker fieldsets

### ISSUE THREAD

**Issue by jmsv6d** [cerner/terra-clinical#435]
# Bug Report
While attempting to resolve accessibility issues for site examples for https://github.com/cerner/terra-ui/issues/46 the onset picker examples were not accessible as the fieldsets have no name associated to them available to accessibility APIs. Consumers should have the ability to add a legend for onset picker field sets for accessibility.

**Comment by jmsv6d** [comment]
Duplicate #461
